.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+language: ruby
+cache: bundler
+sudo: false
+bundler_args: --jobs 4 --retry 2 --without packaging documentation
+before_install:
+  - rm -f Gemfile.lock
```

Comment:
Why are we including `Gemfile.lock` but removing it at this step?

Reply:
The `Gemfile.lock` is not included in this PR and is not currently included in the git repo.

Reply:
Sorry, of course, my mistake. I will blame it on a lack of coffee ;)